### PR TITLE
[zh-cn]Sync  persistent-volume-v1.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Sync upstrem
Preview：https://deploy-preview-36246--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1/
```yaml
content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md
```

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md [hunk 1]
```diff
@@ -660,41 +660,82 @@ PersistentVolumeSpec 是持久卷的规约。
...
+      name is unique within a namespace to reference a secret resource.
+
+    - **csi.nodePublishSecretRef.namespace** (string)
+
+      namespace defines the space within which the secret name must be unique.
+-->
+  - **csi.nodePublishSecretRef** (SecretReference)
+
+    nodePublishSecretRef 是对包含敏感信息的 Secret 对象的引用，
+    从而传递到 CSI 驱动以完成 CSI NodePublishVolume 和 NodeUnpublishVolume 调用。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:723]: how about this ? “以传递到” or “该对象将传递到”
```suggestion
    以传递到 CSI 驱动以完成 CSI NodePublishVolume 和 NodeUnpublishVolume 调用。
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md [hunk 2]
```diff
@@ -660,41 +660,82 @@ PersistentVolumeSpec 是持久卷的规约。
...
+    <a name="SecretReference"></a>
+    **SecretReference 表示对某 Secret 的引用，其中包含足够的信息来访问任何名字空间中的 Secret。**
+
+    - **csi.nodePublishSecretRef.name** (string)
+
+      name 在名字空间内是唯一的，以引用一个 Secret 资源。
+
+    - **csi.nodePublishSecretRef.namespace** (string)
+
+      name 在名字空间内是唯一的，以引用一个 Secret 资源。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:736]: ctrl+c/v? 🤣 
```suggestion
      namespace 定义了 Secret 名称必须唯一的空间。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md [hunk 3]
```diff
@@ -660,41 +660,82 @@ PersistentVolumeSpec 是持久卷的规约。
...
+    - **csi.nodeExpandSecretRef.name** (string)
 
       name 在名字空间内是唯一的，以引用一个 Secret 资源。
     
-    - **csi.nodePublishSecretRef.namespace** (string)
+    - **csi.nodeExpandSecretRef.namespace** (string)
 
       namespace 指定一个名字空间，Secret 名称在该名字空间中必须唯一。
+
+<!--
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:704]:
```suggestion
  <!--
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md [hunk 4]
```diff
@@ -660,41 +660,82 @@ PersistentVolumeSpec 是持久卷的规约。
...
+    *SecretReference represents a Secret Reference. It has enough information to retrieve secret in any namespace*
+
+    - **csi.nodePublishSecretRef.name** (string)
+
+      name is unique within a namespace to reference a secret resource.
+
+    - **csi.nodePublishSecretRef.namespace** (string)
+
+      namespace defines the space within which the secret name must be unique.
+-->
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:719]:
```suggestion
  -->
```

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md [hunk 5]
```diff
@@ -660,41 +660,82 @@ PersistentVolumeSpec 是持久卷的规约。
...
+  - **csi.nodePublishSecretRef** (SecretReference)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:720]: 720 行之前加空行

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md [hunk 6]
```diff
@@ -660,41 +660,82 @@ PersistentVolumeSpec 是持久卷的规约。
...
+
+    nodePublishSecretRef 是对包含敏感信息的 Secret 对象的引用，
+    以传递到 CSI 驱动以完成 CSI NodePublishVolume 和 NodeUnpublishVolume 调用。
+    此字段是可选的，且如果不需要 Secret，则此字段可以为空。
+    如果 Secret 对象包含多个 Secret，则所有 Secret 被传递。
+
+    <a name="SecretReference"></a>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:727]: 727 行删除

File: content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md [hunk 7]
```diff
@@ -660,41 +660,81 @@ PersistentVolumeSpec 是持久卷的规约。
...
+
+    - **csi.nodePublishSecretRef.name** (string)
+
+      name is unique within a namespace to reference a secret resource.
+
+    - **csi.nodePublishSecretRef.namespace** (string)
+
+      namespace defines the space within which the secret name must be unique.
+  -->
+    - **csi.nodePublishSecretRef** (SecretReference)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:720]: 720 行要保持原来的缩进，并且前面要加一个空行（markdown 解析限制）。

**Comment by tengqm** [comment]
Preview 不正确

image

**Comment by yanrongshi** [comment]
> Preview 不正确
> 
> image

这种的我可以理解为正确的吧
image

**Comment by windsonsea** [comment]
The old zh-cn text also has some alignment error:

image

**Comment by yanrongshi** [comment]
Due to some unknown issues, my local fix could not be submitted to the cloud, so I resubmitted a PR about: #36347

**Pull request closed without merging**